bigquery/table.go

```diff
@@ -380,6 +383,66 @@ func bqToMaterializedViewDefinition(q *bq.MaterializedViewDefinition) *Materiali
 	}
 }
 
+// Supported
+type BigLakeFileFormat string
```

Comment:
I wonder if those two new types should be declared separately and have their own comment to avoid the vet issues.

Reply:
Done.